raw_data/preprocessed/north_america/haiti_french.txt

```diff
@@ -0,0 +1,407 @@
+
+(54) L'Agence Nationale des Aires Protégées (ANAP) a pour fonction: [1] de gérer et de coordonner le Système National des Aires Protégées; [2] de coordonner la préparation et la mise en oeuvre du Plan National de Gestion des Aires Protégées; [3] de protéger la diversité biologique in situ et ex situ; [4] d'élaborer ou d'approuver les plans d'aménagement des aires protégées des catégories relevant de sa juridiction et suivre leur implantation; [5] d'étudier les espèces animales et végétales des catégories d'aires protégées relevant de sa juridiction et réaliser des inventaires de flore et de faune; [6] de préserver les aires sous son administration ainsi que celles sous co-gestion; [7] d'élaborer les règlements d'accès aux aires protégées sous sa juridiction et aux ressources bio-génétiques et en autoriser l'accès; [8] d'intégrer, de manière responsable, les populations et les collectivités territoriales dans la gestion des aires protégées sous sa juridiction.
+
+(55) Les procédures, structures et modalités de fonctionnement de l'ANAP sont définies par la Loi.
+
+CHAPITRE IV - DE L'ÉVALUATION ENVIRONNEMENTALE
+
+(56) Les politiques, plans, programmes, projets ou activités susceptibles d'avoir un impact sur l'environnement doivent obligatoirement faire l'objet d'une évaluation environnementale à charge de l'institution concernée. Le processus d'évaluation environnementale couvre l'Étude d'Impact Environnemental (EIE), la déclaration d'impact environnemental, le permis environnemental et les audits environnementaux.
+
+(57) La liste des projets et activités devant faire l'objet d'évaluation environnementale ainsi que les normes et procédures relatives à la mise en route des Études d'Impact Environnemental (EIE) sont établies par voie réglementaire à la charge du Ministère de l'Environnement.
+
+(58) La déclaration d'impact environnemental est soumise, par la personne intéressée, à la non-objection du Ministère de l'Environnement selon les procédures établies par ce dernier. De telles procédures tiendront compte en particulier de la nécessité d'institutionnaliser les audiences publiques en vue d'assurer la plus large participation de la population.
+
+(59) La non-objection environnementale est délivrée par le Ministère de l'Environnement pour les projets et activités qui requièrent une évaluation d'impact environnemental.
+
+(60) Les UTES ont pour obligation d'acheminer, par voie célère au régistre du Ministère de l'Environnement, une notification de tous les processus d'examen de dossiers enclenchés dans le cadre d'évaluations environnementales et une copie de tous les avis émis. Pour chaque document transmis, iI leur sera délivré un numéro d'enregistrement à inscrire dans leur dossier. En cas de contestation ou pour les besoins d'évaluation des performances du système, le Service d'Inspection Générale de l'Environnement du Ministère de l'Environnement peut intervenir et réviser un ou des cas traités par une UTES.
```

Comment:
Microsoft word spellcheck recommends that "régistre" be "registre". I'm not sure which is correct here.

Reply:
`régistre` is an error. It should read `registre`. I will update.